### PR TITLE
Fixes #20282 - Provide more secure defaults for apache

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: config/foreman.hiera/security.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+apache::mod::ssl::ssl_cipher: 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA3'
+apache::trace_enable: 'Off'
```
Comment (ekohl) [inline config/foreman.hiera/security.yaml:3]: You could close http://projects.theforeman.org/issues/14667 with resolved if you apply this.

**Review by ekohl — APPROVED** [review]

File: config/foreman.hiera/security.yaml (hunk 1 above)
Comment (tbrisker) [reply config/foreman.hiera/security.yaml:3]: Updated the commit message to reflect that.

Comment (tbrisker) [reply config/foreman.hiera/security.yaml:3]: not that this is a real security issue, but there is no harm in disabling it and lots of automated security audits flag this as an issue

**Comment by tbrisker** [comment]
Pushed a new commit with expanded message explaining the settings.

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
JFTR: working fine in my setup 👍

**Comment by ekohl** [comment]
Thanks!

**Comment by mmoll** [comment]
This is breaking reporting with PC1 packages on Debian/jessie, even with JAVA 8 from jessie-backports... Could this be postponed to 1.17, where Debian/jessie dropping is on the schedule anyway?

**Comment by ekohl** [comment]
Maybe we could ship with just the trace and comment the stronger ciphers in 1.16 or until we drop Jessie.

**Pull request merged**